moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -167,15 +167,6 @@ void CurrentStateMonitor::startStateMonitor(const std::string& joint_states_topi
     }
     if (tf_buffer_ && !robot_model_->getMultiDOFJointModels().empty())
     {
-      // If a dedicated thread is enabled for the buffer this probably means the user is adding them either through
-      // tf2_ros::TransformListener or themselves, so we print a warning message warning that transformCallback is doing
-      // the same duplicate operation
-      if (tf_buffer_->isUsingDedicatedThread())
-      {
-        RCLCPP_WARN(LOGGER, "The tf2_ros::Buffer is attached to tf2_ros::TransformListener and the internal tf "
-                            "subscribers inside CurrentStateMonitor, you may want to remove the transform listener to "
-                            "avoid duplicate addition to the same transforms");
```

Comment:
This is still a very valid warning when CSM is used *outside* PSM.
There was probably a reason why this was added in the first place and I would be grateful for the feedback in such a case.

Reply:
I agree, we should keep the warning

Reply:
I added this warning in the last PR to warn about the previous behavior of using an external tf listener, but since we have an internal one now it will always print this warning which may confuse users

Reply:
Hmm, so in both cases (PSM+CSM, CSM standalone) we use a dedicated thread for the tf_buffer, right? I guess there is no straight-forward way to check for external listeners anymore. It's fine to remove this warning then, IMO.
Thoughts for a follow-up: Since we don't want external listeners to run at the same time (only PSM is allowed), how about we make the current CSM constructors friend constructors for PSM and only provide public versions without tf_buffer?